exec-unittests

```diff
@@ -186,8 +186,12 @@ for _major in ${major_list}; do
         if [[ ${?} -ne 0 ]]; then
             rc=1
         fi
+        if [[ ${count} -eq 0 && "${subtst}" != "python" ]]; then
```

Comment:
Since `count` is only referenced in the block starting at line 172, should its initialization be moved into that block?